--- kdiff3-0.9.88/src/kdiff3.desktop.orig	2005-02-22 08:39:31.000000000 +0100
+++ kdiff3-0.9.88/src/kdiff3.desktop	2005-02-25 15:53:42.920158984 +0100
@@ -1,11 +1,9 @@
-
 [Desktop Entry]
 Encoding=UTF-8
 Name=KDiff3
 Name[hi]=के-डिफ3
 Name[sv]=Kdiff3
 Name[ta]=கேடிஃப்3
-Name[xx]=xxKDiff3xx
 GenericName=Diff/Patch Frontend
 GenericName[bs]=Interfejs za Diff/Patch
 GenericName[cy]=Blaen Gwahaniaethau/Clytiau
@@ -31,7 +29,6 @@
 GenericName[ta]= Diff/Patch Frontend
 GenericName[tg]=Интерфейси графикӣ ба Diff ва Patch
 GenericName[uk]=Інтерфейс до diff/patch
-GenericName[xx]=xxDiff/Patch Frontendxx
 GenericName[zh_CN]=Diff/Patch 前端
 GenericName[zu]=Diff/PatchIsiqalo sokugcina
 Exec=kdiff3 %i %m -caption "%c"
@@ -63,6 +60,7 @@
 Comment[sv]=Ett jämförelseverktyg för fil- och katalogjämförelser
 Comment[tg]=Асбоби баробаркунӣ ва пайванди файлҳо ва каталогҳо
 Comment[uk]=Засіб-утиліта для порівняння і поєднання файлів та каталогів
-Comment[xx]=xxA File And Directory Comparison And Merge Toolxx
 Comment[zh_CN]=一个文件和目录的比较及合并工具
 Terminal=false
+Categories=Qt;KDE;Utility;
+# vi: encoding=utf-8
