diff -urN kdiff3-0.9.88.org/admin/cvs.sh kdiff3-0.9.88/admin/cvs.sh
--- kdiff3-0.9.88.org/admin/cvs.sh	2004-09-13 21:21:55.000000000 +0200
+++ kdiff3-0.9.88/admin/cvs.sh	2006-04-01 17:12:23.803626500 +0200
@@ -64,7 +64,7 @@
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8.* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9.* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
diff -urN kdiff3-0.9.88.org/doc/Makefile.am kdiff3-0.9.88/doc/Makefile.am
--- kdiff3-0.9.88.org/doc/Makefile.am	2005-02-24 23:32:57.000000000 +0100
+++ kdiff3-0.9.88/doc/Makefile.am	2006-04-01 17:11:54.981825250 +0200
@@ -1,6 +1,6 @@
 # the SUBDIRS is filled automatically by am_edit. If files are 
 # in this directory they are installed into the english dir
 
-KDE_LANG = en da de et fr it pt sv
+SUBDIRS = en da de et fr it pt sv
 KDE_DOCS = kdiff3
-SUBDIRS = $(AUTODIRS)  
+#SUBDIRS = $(AUTODIRS)  
