--- kdiff3-0.9.80/src/diff.cpp.orig	2003-11-15 22:48:25.000000000 +0000
+++ kdiff3-0.9.80/src/diff.cpp	2004-01-07 17:33:28.000000000 +0000
@@ -127,8 +127,8 @@
       int size1=pLd1->size;
       int size2=pLd2->size;
 
-      int i1=min2(pLd1->pFirstNonWhiteChar - pLd1->pLine,size1);
-      int i2=min2(pLd2->pFirstNonWhiteChar - pLd2->pLine,size2);
+      int i1=min2(pLd1->pFirstNonWhiteChar - pLd1->pLine,(ptrdiff_t)size1);
+      int i2=min2(pLd2->pFirstNonWhiteChar - pLd2->pLine,(ptrdiff_t)size2);
       for(;;)
       {
          while( i1<size1 && isWhite( p1[i1] ) ) ++i1;
